testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeColumnMappingMode.java

```diff
@@ -485,4 +483,138 @@ private void testSpecialCharacterColumnNamesWithColumnMappingMode(String mode)
             onDelta().executeQuery("DROP TABLE default." + tableName);
         }
     }
+
+    @Test(groups = {DELTA_LAKE_DATABRICKS, DELTA_LAKE_EXCLUDE_73, DELTA_LAKE_EXCLUDE_91, PROFILE_SPECIFIC_TESTS})
+    @Flaky(issue = DATABRICKS_COMMUNICATION_FAILURE_ISSUE, match = DATABRICKS_COMMUNICATION_FAILURE_MATCH)
+    public void testTrinoDropColumnWithColumnMappingModeId()
+    {
+        testTrinoDropColumnWithColumnMappingModeName("id");
+    }
+
+    @Test(groups = {DELTA_LAKE_DATABRICKS, DELTA_LAKE_OSS, DELTA_LAKE_EXCLUDE_73, DELTA_LAKE_EXCLUDE_91, PROFILE_SPECIFIC_TESTS})
```

Comment:
According to https://docs.databricks.com/sql/language-manual/sql-ref-syntax-ddl-alter-table.html

```
DROP COLUMN

Preview

This feature is in [Public Preview](https://docs.databricks.com/release-notes/release-types.html).

Applies to: check marked yes Databricks SQL check marked yes Databricks Runtime 11.0 and above
```